src/config_generator/from_openapi.rs

```diff
@@ -0,0 +1,209 @@
+use std::collections::BTreeMap;
+
+use oas3::spec::ObjectOrReference;
+use oas3::{Schema, Spec};
+
+use crate::config::{Arg, Config, Field, Http, RootSchema, Server, Type, Upstream};
+use crate::http::Method;
+
+trait ToDebugString: std::fmt::Debug {
+    fn to_debug_string(&self) -> String {
+        format!("{:?}", self)
+    }
+}
+
+impl<T: std::fmt::Debug> ToDebugString for T {}
```

Comment:
- Call it something else, display or something else.
- Implement it for specific types of only open API entities.